t/python/t0010-job.py

```diff
@@ -396,12 +396,82 @@ def test_22_from_batch_command(self):
                 self.fh,
                 JobspecV1.from_batch_command("sleep 0", "nested sleep with no shebang"),
             )
+        # extra parameters are accepted:
+        environment = {"TEST": 1, "FOO": "BAR"}
+        env_expand = {"EXPAND": "{{tmpdir}}"}
+        rlimits = {"nofile": 12000}
+
+        jobspec = JobspecV1.from_batch_command(
+            "#!/bin/sh\0sleep0",
+            "testjob",
+            environment=environment,
+            env_expand=env_expand,
+            rlimits=rlimits,
+            duration=60.0,
+            cwd="/my/cwd",
+            stdout="output-file",
+            stderr="error-file",
+            label_io=True,
+            unbuffered=True,
+            queue="testq",
+            bank="testbank",
+        )
+        self.assertDictEqual(jobspec.environment, environment)
+        self.assertDictEqual(
+            jobspec.getattr("system.shell.options.env-expand"), env_expand
+        )
+        self.assertDictEqual(jobspec.getattr("system.shell.options.rlimit"), rlimits)
+        self.assertEqual(jobspec.duration, 60)
+        self.assertEqual(jobspec.cwd, "/my/cwd")
+        self.assertEqual(jobspec.stdout, "output-file")
+        self.assertEqual(jobspec.stderr, "error-file")
+        self.assertTrue(jobspec.getattr("system.shell.options.output.stdout.label"))
+        self.assertTrue(jobspec.getattr("system.shell.options.output.stderr.label"))
+        self.assertEqual(
+            jobspec.getattr("system.shell.options.output.stdout.buffer.type"),
+            "none",
+        )
+        self.assertEqual(jobspec.getattr("system.queue"), "testq")
+        self.assertEqual(jobspec.getattr("system.bank"), "testbank")
 
     def test_23_from_nest_command(self):
         """Test that `from_batch_command` produces a valid jobspec"""
```

Comment:
Next door I realize but I assume this should be `from_nest_command`?
```suggestion
        """Test that `from_nest_command` produces a valid jobspec"""
```

Reply:
Good catch! I'll add a fix for that.